test/common/router/config_impl_test.cc

```diff
@@ -3677,6 +3677,67 @@ name: RequestMirrorPoliciesLevels
   EXPECT_EQ("rc_cluster", rc_policies[0]->cluster());
 }
 
+
+TEST_F(RouteMatcherTest, RequestMirrorPoliciesVirtualHost) {
+  const std::string yaml = R"EOF(
+name: RequestMirrorPoliciesVirtualHost
+virtual_hosts:
+- name: www2
+  request_mirror_policies:
+    - cluster: vh_cluster
+  domains:
+  - www.lyft.com
+  routes:
+  - match:
+      prefix: "/foo"
+    route:
+      request_mirror_policies:
+        - cluster: route_cluster
+      cluster: www2
+  - match:
+      prefix: "/bar"
+    route:
+      cluster: www2
+- name: www3
```

Comment:
nit: use www2 here as the name to avoid confusion

Reply:
done